### PR TITLE
Fix: set exists:false precondition for deleted docs

### PR DESCRIPTION
**Pull request by ehsannas**

### CODE REVIEW

**Comment by ehsannas** [comment]
well... that didn't go as i expected heh

File: packages/firestore/src/core/transaction.ts [hunk 1]
```diff
@@ -142,12 +142,18 @@ export class Transaction {
 
   /**
    * Returns the version of this document when it was read in this transaction,
-   * as a precondition, or no precondition if it was not read.
+   * as a precondition, or an existence precondition if the document did not
```
Comment (egilmorez) [inline packages/firestore/src/core/transaction.ts:145]: This kind of list would normally only have one "or,"
so we'd have:

". . . as a precondition, an existence precondition if the document did not exist, or . . . ."

**Review by egilmorez — COMMENTED** [review]
Sorry to miss this.  Have one suggestion, thanks!

**Comment by ehsannas** [comment]
closing this in favor of https://github.com/firebase/firebase-js-sdk/pull/6550

**Pull request closed without merging**